tvb_nest/interfaces/models.py

```diff
@@ -1,16 +1,25 @@
 # -*- coding: utf-8 -*-
 
-from tvb_nest.config import CONFIGURED
+from tvb_nest.config import CONFIGURED, initialize_logger
 from tvb_nest.interfaces.base import TVBNESTInterface
-from tvb_scripts.utils.log_error_utils import initialize_logger
 
+from tvb.simulator.models.reduced_wong_wang_exc_io import ReducedWongWangExcIO
 from tvb.simulator.models.reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
-from tvb.simulator.models.wilson_cowan_constraint import WilsonCowan
-from tvb.simulator.models.generic_2d_oscillator_multiscale import Generic2dOscillator
+from tvb.simulator.models.wilson_cowan_constraint import WilsonCowan as WilsonCowanSimModel
+from tvb.simulator.models.generic_2d_oscillator_multiscale import Generic2dOscillator as Generic2dOscillatorSimModel
```

Comment:
Whouldn't it be better if the Cosimulator sublaccs together with these `models.* multiscale ` classes would sit separately of tvb-library ?
Either under `tvb-contrib` or under `tvb-multiscale.core` ?

Reply:
Indeed the multiscale models could be moved to either tvb-contrib or tvb-multiscale.core.tvb.models.
Whether the CoSimulator will also move there will be I guess a decision to make after we decide how to implement the TVB infrastructure for co-simulation, and how much of it should be in the scientific_library of TVB.